src/backends/postgresql/session.cpp

```diff
@@ -29,6 +30,130 @@ void hard_exec(postgresql_session_backend & session_backend,
     postgresql_result(session_backend, PQexec(conn, query)).check_for_errors(errMsg);
 }
 
+// helper function to quote a string before sinding to PostgreSQL
```

Comment:
There is a typo here:
```suggestion
// helper function to quote a string before sending to PostgreSQL
```